src/styles/main.scss

```diff
@@ -323,3 +323,18 @@ div.rbt div {
 .modal {
   background: rgba(0, 0, 0, 0.5);
 }
+
+.rdf-display-panel {
+  background-color: #E8ECF4;
+  border: 1px solid #979797;
+  padding: 1rem;
+  margin-bottom: 1rem;
+}
+
+.rdf-display-panel table th {
+  background-color: #E8ECF4;
+}
+
+.rdf-display-panel table td {
+  background-color: #E8ECF4;
+}
```

Comment:
This forces the background of the table to match the panel, if we want to keep, anyone know the CSS selector format to merge with the above `th` declaration?